plugin/storage/integration/es_index_cleaner_test.go

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build index_cleaner
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/olivere/elastic.v5"
+)
+
+// 1. removes daily indices, but does not remove rollover or archive
```

Comment:
What do these comments relate to? Assume the specific test instances in `TestIndexCleaner` - would be better to move comments to those test instances - and there are three so an additional comment would be required.

Reply:
I forgot to remove these